modules/test/playwright/tests/dynamic-data-mapping-form-web/formEntries.spec.ts

```diff
@@ -30,6 +30,44 @@ test.beforeEach(({page}) => {
 	page.setViewportSize({height: 1080, width: 1920});
 });
 
+test('error reference is displayed for invalid multiple selection', async ({
+	formBuilderPage,
+	formBuilderSidePanelPage,
+	formsPage,
+	page,
+}) => {
+	await formsPage.goTo();
+
+	await formsPage.clickManagementToolbarNewButton();
+
+	await formBuilderSidePanelPage.addMultipleSelectionButton.dblclick();
+
+	await formBuilderSidePanelPage.requiredFieldToggleSwitch.click();
+
+	await page.waitForTimeout(1000);
+
+	await formBuilderPage.clickPublishFormButton();
+
+	const formSubmissionURL = await formBuilderPage.getFormSubmissionURL();
+
+	await page.goto(formSubmissionURL, {waitUntil: 'networkidle'});
+
+	await page.getByRole('button', {name: 'Submit'}).click();
+
+	const singleSelect = page.getByRole('checkbox');
+
+	await singleSelect.waitFor();
+
+	const fieldFeedbackId =
+		await singleSelect.getAttribute('aria-errormessage');
```

Comment:
This is the core of this test, right? if this `getAttribute` does not return something for `fieldFeedbackId`, then the rest of the test will fail and we will investigate and find out that `aria-errormessage` is not being set anymore. This is why I think we should add an specific assertion here. I think that `expect(fieldFeedbackId).toBeTruthy();` would do it.

Reply:
Or maybe https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute

Reply:
In fact, this test goes further. The objective of the test is to identify whether there is a valid reference to the error message value, and this reference must contain the expected content.

The way the test is written, if it fails, we will have no doubts. We will know that there is a problem in this flow.

This suggested expected value would not bring many benefits. We would only know whether it failed due to the lack of a reference or if the reference is invalid. Something that with a few seconds of investigation we would already have the answer.

Reply:
About 3. check aria attributes to see if they exist and have the right value

Checking the existence of the value is not what will bring the expected result to the client, the focus is to check if the reference is valid, and that is what we are doing when we build the fieldFeedbackId based on the aria-errormessage property